sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/V2SessionCatalog.scala

```diff
@@ -156,15 +156,6 @@ class V2SessionCatalog(catalog: SessionCatalog)
       columns: Array[Column],
       partitions: Array[Transform],
       properties: util.Map[String, String]): Table = {
-    createTable(ident, CatalogV2Util.v2ColumnsToStructType(columns), partitions, properties)
-  }
-
-  // TODO: remove it when no tests calling this deprecated method.
-  override def createTable(
```

Comment:
I will move the specific implementation of createTable from `createTable(..., StructType, ...)` to `createTable(..., Array[Column], ...)`, so that when we delete the deprecated method `createTable(..., StructType, ...)` in `TableCatalog`, we can directly delete the method `createTable(..., StructType, ...)` here.